src/site/content/en/reliable/http-cache/index.md

```diff
@@ -118,65 +175,178 @@ ignored](https://www.keycdn.com/blog/cache-control-immutable#browser-support) in
 
 Unfortunately, not all of the URLs you load are versioned. Maybe you're not able
 to include a build step prior to deploying your web app, so you can't add hashes
 to your asset URLs. And every web application needs HTML files—those files are
 (almost!) never going to include versioning information, since no one will
-bother using your web app if they need to remember that the URL to visit is
+bother to use your web app if they need to remember that the URL to visit is
 `https://example.com/index.34def12.html`. So what can you do for those URLs?
 
 This is one scenario in which you need to admit defeat. HTTP caching alone isn't
 powerful enough to avoid the network completely. (Don't worry—you'll soon learn
-about service workers, which will provide the support we need to swing the
-battle back in your favor.) But there are a few steps you can take to make sure
-that network requests are as quick and efficient as possible.
-
-First off, make sure you add `Cache-Control: no-cache` to your response
-messages. This explicitly tells the browser that you've surrendered, and that
-going to the network to check if there's been an update is necessary.
-
-Along with that, setting one of two additional response headers is recommended:
-either
-[`Last-Modified`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified)
-or [`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag).
-ETags are identifiers for specified resources. They allow caches to be more
-efficient and are useful to help prevent simultaneous updates from overwriting
-each other.  By setting one or the other of those headers, you'll end up making
-the revalidation request much more efficient. They end up triggering the
-[`If-Modified-Since`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since)
-or
-[`If-None-Match`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match)
-request headers that we mentioned earlier.
+about [service workers](/service-workers-cache-storage/), which will provide the
+support we need to swing the battle back in your favor.) But there are a few
+steps you can take to make sure that network requests are as quick and efficient
+as possible.
+
+The following `Cache-Control` values can help you fine-tune where and how unversioned URLs
+are cached:
+
+* `no-cache`. This instructs the browser that it must revalidate with the
+  server every time before using a cached version of the URL.
+* `no-store`. This instructs the browser and other intermediate caches (like CDNs) to never
+  store any version of the file. `no-store` is often used for files that have sensitive or
+  personal data.
+* `private`. Browsers can cache the file but intermediate caches cannot.
+  For example, `private` would make sense for a server-side rendered HTML file
+  that contains private user information.
+* `public`. The response can be stored by any cache, even if it's usually not cacheable.
```

Comment:
I'm not clear on what "even if it's usually not cacheable" means.

Reply:
```suggestion
* `public`. The response can be stored by any cache.
```

Good point. I'm also not sure what it means. I think we can just leave it out.